gui/doc/input.csv

```diff
@@ -1,4 +1,6 @@
 ﻿Name,Type,Default,Doc
 *value,dynamic(any),None,The value represented by this control.
+change_delay,int,<i>App config</i>,"minimum time between triggering two <i>on_change</i> callback.
+The default value is defined at the application configuration level by change_delay. if None or 0, the delay is set to 300ms."
```

Comment:
"configuration level by change_delay"
->
<pre>
configuration level by the **change_delay** configuration option
</pre>

I guess that if this was set to 0, I would want no delay at all... Not a (not so) strange default value, that makes a lot of sense if None.

Reply:
is markdown supported in csv ?